socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -61,31 +58,60 @@ export default Vue.component("author-bar", {
         isUserAuthentificated: {type: Boolean, required: true},
     },
     data() {
-        return {showProfileBox: false}
+        return {
+            showProfileBox: false,
+            _isUserFollowingAuthor: this.isUserFollowingAuthor,
+        }
     },
     computed: {
+        isUserRemote(){
+            return !this.isUserLocal
+        },
         showFollowBtn() {
-            return this.isUserAuthentificated
-                    && !this.isUserLocal
-                    && !this.isUserFollowingAuthor
+            return this.canFollow && !this.$data._isUserFollowingAuthor
         },
         showUnfollowBtn() {
+            return this.canFollow && this.$data._isUserFollowingAuthor
+        },
+        canFollow() {
             return this.isUserAuthentificated
-                    && !this.isUserLocal
-                    && this.isUserFollowingAuthor
+                && !this.isUserAuthor
+                && this.isUserLocal
```

Comment:
Wait hmm why do we check if the user is local? Who is `isUserLocal`, the author or the logged in user? Anyway, remote users can be followed too.